app/views/simulations/new.html.erb

```diff
@@ -5,5 +5,5 @@
     <%= f.label :runs %>
     <%= f.number_field :runs %>
 
-    <%= f.submit %>
+    <%= f.submit class: "bg-blue-800 text-white px-5 py-2 rounded-full hover:bg-blue-400"%>
```

Comment:
If you want it, here's some button code from the paid TailwindUI component library. I paid for the lifetime license when I was building Leevo.

```html
<button type="button" class="rounded-md bg-indigo-600 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">Button text</button>

```

Reply:
Wow!!!! That is a ton of tags. Added it to all buttons at #21 

Used the helper to keep the CSS tags.